cpu/pic32/lib/pic32_i2c.c

```diff
@@ -78,6 +78,12 @@
     return 0;                                                            \
   }                                                                      \
   uint8_t                                                                \
+  i2c##XX##_release(void)                                                \
```

Comment:
Is this just for a consistent API, as the current init function does this anyway? Perhaps if adding this we should force its use by changing the init function to assert that it is 0 rather than setting 0?

Therefore expecting this sequence to fail: `i2c##XX##_init, i2c##XX##_init`
But this sequence to work: `i2c##XX##_init, i2c##XX##_release, i2c##XX##_init`